src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,23 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null || storageDao.get(user.getLogin()) != null) {
```

Comment:
let's move this `storageDao.get(user.getLogin()) != null` to separate check and write a specific message that this user already exists